src/cryptsetup/cryptsetup-generator.c

```diff
@@ -854,14 +854,14 @@ static int add_crypttab_devices(void) {
                 return 0;
         }
 
-        for (;;) {
+        for (r = 0;;) {
                 _cleanup_free_ char *line = NULL, *name = NULL, *device = NULL, *keyspec = NULL, *options = NULL;
                 int k;
 
-                r = read_stripped_line(f, LONG_LINE_MAX, &line);
-                if (r < 0)
+                k = read_stripped_line(f, LONG_LINE_MAX, &line);
+                if (k < 0)
                         return log_error_errno(r, "Failed to read %s: %m", arg_crypttab);
```

Comment:
`r` -> `k`

But, typically, `r` is used for storing short-lived error code. So, I think it is better to use `int ret;` for gathering results in the below, and use `r` for `read_stripped_line()`.

Reply:
Okay.